### PR TITLE
VM service docs improvements

### PR DESCRIPTION
**Pull request by rosieyohannan**
fixes: #4652 

I've cleared up the language a bit, removed a duplicate section and added markers to the screenshot so it's clear what each section of the doc is referring to:

Screenshot 2020-09-16 at 16 14 54

### CODE REVIEW

File: jekyll/_cci2/vm-service.adoc [hunk 1]
```diff
@@ -104,7 +99,7 @@ Jobs run using the remote Docker Environment, or the `machine` executor are sche
 
 // add steps to find out statuses of Remote Docker and machine executor - would this be run `nomad node-status` from one of the nomad client instances? If so, how does a user navigate to a Nomad client instance?
 
-NOTE: A cron job in scheduled to cycle all default and preallocated instanced at least once per day to ensure instanced don't end up in a dead/bad state.
+NOTE: A cron job is scheduled to cycle all default and preallocated instances at least once per day to ensure instances do not end up in a unworkable state.
```
Comment (teesloane) [inline jekyll/_cci2/vm-service.adoc:102]:
```suggestion
NOTE: A cron job is scheduled to cycle all default and preallocated instances at least once per day to ensure instances do not end up in an unworkable state.
```

File: jekyll/_cci2/vm-service.adoc [hunk 2]
```diff
@@ -66,20 +66,15 @@ image::vm-service-custom.png[Custom VM Service Images]
 * Packer (https://packer.io/intro/getting-started/install.html)
 * AWS Access Key ID and Secret Access Key
 
-==== Creating a Custom AMI
+==== Creating a Custom Linux AMI
 
 1. Clone our image builder repo: https://github.com/circleci/image-builder/tree/picard-vm-image
```
Comment (teesloane) [inline jekyll/_cci2/vm-service.adoc:71]: I know you didn't make a change to this line, but could we make this an inline link?

**Review by teesloane — CHANGES_REQUESTED** [review]
Nice! Just make a suggestion or two.

File: jekyll/_cci2/vm-service.adoc (hunk 2 above)
Comment (rosieyohannan) [reply jekyll/_cci2/vm-service.adoc:71]: On the site it is a link:
Screenshot 2020-09-16 at 21 19 28

or do you mean [image builder repo](URL) style?

Comment (rosieyohannan) [reply jekyll/_cci2/vm-service.adoc:71]: @teesloane ^

Comment (rosieyohannan) [reply jekyll/_cci2/vm-service.adoc:71]: Thanks for the review BTW!

**Review by teesloane — APPROVED** [review]

**Pull request merged**